tests/acceptance/request_test.py

```diff
@@ -99,6 +114,7 @@ def test_400_if_required_body_is_missing(test_app):
     ).status_code == 400
 
 
+@pytest.mark.xfail
```

Comment:
Works for me
On Fri, Mar 27, 2015 at 2:58 PM Daniel Nephin <notifications@github.com>
wrote:

> In tests/acceptance/request_test.py
> <https://github.com/striglia/pyramid_swagger/pull/93#discussion_r27334058>
> :
>
> > @@ -99,6 +114,7 @@ def test_400_if_required_body_is_missing(test_app):
> >      ).status_code == 400
> >
> >
> > +@pytest.mark.xfail
>
> We could just treat "no content-type" as json, and only read request.POST
> if the header is one of 'application/x-www-form-urlencoded',
> 'multipart/form-data'
>
> I believe a proper content type is going to be required for that anyway,
> right?
>
> So form could be
>
> @property def form(self):
>     if self.request.headers.get('Content-Type') in FORM_TYPES:
>        return request.POST
>     return {}
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/striglia/pyramid_swagger/pull/93/files#r27334058>.
>